src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -180,26 +180,21 @@ static RadialGradient* _applyRadialGradientProperty(SvgStyleGradient* g, const B
 }
 
 
-//The SVG standard allows only for 'use' nodes that point directly to a basic shape.
-static bool _appendClipUseNode(SvgLoaderData& loaderData, SvgNode* node, Shape* shape, const Box& vBox, const string& svgPath)
+static bool _appendClipChild(SvgLoaderData& loaderData, SvgNode* node, Shape* shape, const Box& vBox, const string& svgPath)
 {
-    if (node->child.count != 1) return false;
-    auto child = *(node->child.data);
-
-    Matrix finalTransform = {1, 0, 0, 0, 1, 0, 0, 0, 1};
-    if (node->transform) finalTransform = *node->transform;
-    if (node->node.use.x != 0.0f || node->node.use.y != 0.0f) {
-        finalTransform *= {1, 0, node->node.use.x, 0, 1, node->node.use.y, 0, 0, 1};
-    }
-    if (child->transform) finalTransform = *child->transform * finalTransform;
-
-    return _appendClipShape(loaderData, child, shape, vBox, svgPath, identity((const Matrix*)(&finalTransform)) ? nullptr : &finalTransform);
-}
+    //The SVG standard allows only for 'use' nodes that point directly to a basic shape.
+    if (node->type == SvgNodeType::Use) {
+        auto child = *(node->child.data);
```

Comment:
if the use node references a node that doesn;t exist, a crash will occure:
```
<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
  
<clipPath id="clip">
    <use href="#m1" x="0" fill="blue" />
</clipPath>

<rect x="0" y="0" width="100" height="100"  fill="red" clip-path="url(#clip)"
      fill-opacity="0.5"/>

</svg>
```

I removed from the sample:
```
  <defs>
    <circle id="m1" cx="30" cy="50" r="30" transform="matrix(1 0.44 1 1 0 10)"/>
  </defs>
```